applications/RomApplication/RomApplication.py

```diff
@@ -1,4 +1,3 @@
-

```

Comment:
Very minor, but I'd try to introduce these accidental changes in the history of the file. To revert it to the master version you can to `git checkout master RomApplication.py`.